src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/CodeActions/Razor/ExtractToComponentCodeActionProvider.cs

```diff
@@ -56,7 +55,18 @@ public Task<ImmutableArray<RazorVSInternalCodeAction>> ProvideAsync(RazorCodeAct
             return SpecializedTasks.EmptyImmutableArray<RazorVSInternalCodeAction>();
         }
 
-        var actionParams = CreateActionParams(startNode, endNode, @namespace);
+        var span = TryGetSpanFromNodes(startNode, endNode, context);
+        if (span is null)
```

Comment:
Nit: To save two calls to `.Value` below, make this `if (span is { } textSpan)`, or whatever name you feel is best.